src/material-experimental/mdc-core/option/option.scss

```diff
@@ -0,0 +1,58 @@
+@import '@material/list/mixins.import';
+@import '@material/list/variables.import';
+@import '../../mdc-helpers/mdc-helpers';
+@import '../../../material/core/style/vendor-prefixes';
+@import '../../../cdk/a11y/a11y';
+
+.mat-mdc-option {
+  // Note that we include this private mixin, because the public
+  // one adds a bunch of styles that we aren't using for the menu.
```

Comment:
For a follow-up: It might be good capturing things we actually need. If we come back to this in the future, we don't know what is actually needed / what the public mixin currently provides that we don't need.